packages/e2e/features/ui/components/authenticator/verify-user.feature

```diff
@@ -19,7 +19,7 @@ Feature: Verify User
     Then "New Label" field does not have class "amplify-visually-hidden"
     Then I see placeholder "Enter your Confirmation Code:"
 
-  @react-nativee
+  @todo-react-native
```

Comment:
This tag was not meant as typo but tracked a follow-up work bc this test suite uses sign-in example app and mocks one API on web, but that is not possible with Detox (all apis or no apis get mocked) so this needs a separate example app to be able to run in RN.
Fix uninspired and confusing tag name.

Reply:
Should we add a comment on why we are skipping it?